src/js/dropdown-menu/dropdown.underscore

```diff
@@ -7,12 +7,19 @@
     <% } %>
     <%- main.text %>
 </a>
-<button type="button" class="button button-more has-dropdown js-dropdown-button" aria-haspopup="true" aria-expanded="false" aria-controls="edx-user-menu">
-    <span class="sr-only"><%- button_label %></span>
+<button type="button" class="menu-button button-more has-dropdown js-dropdown-button <% if (!button.icon) { %>default-icon<% } %>" aria-haspopup="true" aria-expanded="false" aria-controls="<%- menuId %>">
+    <% if (button.icon) { %>
+        <span class="icon-fallback icon-fallback-img">
+            <span class="icon <%- button.icon %>" aria-hidden="true"></span>
+            <span class="text"><%- button.label %></span>
```

Comment:
Because a pseudo element generates a caret for non-icons. The button label is still hidden for line 14, but using the icon syntax from the pattern library.